components/camel-bindy/pom.xml

```diff
@@ -40,6 +41,11 @@
             <groupId>org.apache.camel</groupId>
             <artifactId>camel-support</artifactId>
         </dependency>
+ 		<dependency>
+			<groupId>com.ibm.icu</groupId>
+			<artifactId>icu4j</artifactId>
```

Comment:
Using this will require adding a license for this.

https://github.com/search?q=org%3Aapache+%27com.ibm.icu.icu4j%27&type=Code

Other projects are already using it, not sure this is a good idea, by the way.